test/server/util/twilio_mocks.js

```diff
@@ -54,19 +60,26 @@ export function createCall() {
   // Allie's create call callback Object goes here
 }
 
+function sliceEndpoint(url) {
+  return url.slice(url.length - 1 - 6);
+}
+
 export const testTwilioClient = {
   calls(callSid) {
     return {
       update: (options) => {
         const optionsCalled = Object.getOwnPropertyNames(options).sort();
-        if (lodash.isEqual(optionsCalled, [ 'status', 'statusCallback' ])) {
+        if (lodash.isEqual(optionsCalled, ['status', 'statusCallback'])) {
```

Comment:
We'll have to change this logic in the mock as well since we're not specifying a new statusCallback url for the terminating the contact call.

Reply:
Yeah, that actually will make this test mock helper a bit cleaner, the function I wrote to distinguish between the two .update calls was a bit hacky.